Gemfile

```diff
@@ -2,60 +2,41 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.5.3'
+gem 'bcrypt'
```

Comment:
Hey! Will we use `bcrypt` as BE throughout this project? For FE at least, we've rescinded the manual password and dependent on Google Auth :) Maybe we're hashing something towards Microservices?

Reply:
Oh I see nvm it was a gem from when V and I initially worked on BE, we'll remove it once it gets merged :)